onnx/backend/test/case/base.py

```diff
@@ -8,8 +8,9 @@
 import os
 import sys
 from textwrap import dedent
+from typing import Dict, Text, List, Tuple
 
-import numpy as np
+import numpy as np  # type: ignore
```

Comment:
Is there a way to avoid this? Need to mark every numpy import is pretty cumbersome.

Reply:
I could set `ignore_missing_imports = True` in the config file. However, that would mean that mypy won't report any invalid imports anymore (i.e. not just missing typing information, but it'll also not show any warning if the whole module is missing). And if for some reason a module exists but mypy doesn't find it, it'll also silently ignore it and not typecheck anything related to that module, hiding a lot of errors. Not sure if that is a good idea.

Reply:
An alternative might be to have a local onnx.numpy module that just does

    from numpy import *  # type: ignore

Then, other modules could do

    from onnx import numpy

without having to add `type: ignore`

Reply:
@bddppq Opinions?

Reply:
I'm a little scared of playing with python import mechanism :-), let's not add a local numpy import proxy.

Reply:
@smessmer 
BTW, what does it ignore? Imported but not used?